docs/config.toml

```diff
@@ -2,74 +2,563 @@ baseurl = "https://docs.helm.sh/"
 languageCode = "en-us"
 title = "Helm Docs"
 theme = "helmdocs"
+
+contentDir = "source/docs"
 publishdir = "app"
 
 [params]
     title = "Helm Docs"
     author = "Ronan Flynn-Curran"
     description = "Documentation for Helm - The Kubernetes Package Manager."
 
-canonifyURLs = "false"
-relativeURLs = "true"
+canonifyURLs = "true"
+relativeURLs = "false"
 pluralizeListTitles = "false"
 plainIdAnchors = "true"
+defaultContentLanguage = "en"
+
 
 [[menu.main]]
+  name = "Home"
+  url = "https://docs.helm.sh"
```

Comment:
Opened this as a follow-up issue ([#49](https://github.com/helm/helm-www/issues/49)). 
Hugo makes it a bit tricky, but it should be doable.